Podverse/Podcast.swift

```diff
@@ -36,9 +34,18 @@ class Podcast: NSManagedObject {
         self.mutableSetValue(forKey: "episodes").remove(value)
     }
     
-    static func retrieveMediaRefsFromServer(episodeMediaUrl: String? = nil, podcastFeedUrl: String? = nil, onlySubscribed: Bool? = nil, completion: @escaping (_ mediaRefs:[MediaRef]?) -> Void) {
+    static func retrievePodcastByFeedUrl(feedUrlString: String) -> Podcast? {
```

Comment:
Would you mind changing the name of the method actually? Thats what threw me off I think. Something like fetchPodcastWithFeedUrl(feedURLString:String)